docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -1072,212 +1072,621 @@ thor:
-              "target:thor".</para>
-            </listitem>
+        <para>The pods: [list] item can contain one of the following:</para>
 
-            <listitem>
-              <para>Pod, "Deployment" metadata name from the name of the array
-              item of a type. For example, "eclwatch", "mydali",
-              "thor-thoragent"</para>
-            </listitem>
+        <informaltable colsep="1" frame="all" rowsep="1">
+          <tgroup cols="2">
+            <colspec colwidth="125.55pt" />
 
-            <listitem>
-              <para>Job name regular expression: For example "compile-" or
-              "compile-." or exact match "^compile-.$"</para>
-            </listitem>
+            <colspec />
 
-            <listitem>
-              <para>All: to apply for all HPCC Systems components. The default
-              placements for pods we deliver is [all]</para>
-            </listitem>
-          </orderedlist>
-
-          <para><emphasis role="bold">Placements</emphasis> – in Kubernetes
-          the Placement concept allows you to spread your pods across types of
-          nodes with particular characteristics. Placements would be used to
-          ensure that pods or jobs that want nodes with specific
-          characteristics are placed on them.</para>
-
-          <para>For instance a Thor cluster could be targeted for machine
-          learning using nodes with a GPU. Another job may want nodes with a
-          good amount more memory or another for more CPU. You can use
-          placements to ensure that pods with specific requirements are placed
-          on appropriate nodes.</para>
+            <tbody>
+              <row>
+                <entry>Type: &lt;component&gt;</entry>
+
+                <entry>Covers all pods/jobs under this type of component. This
+                is commonly used for HPCC Systems components. For example, the
+                <emphasis>type:thor</emphasis> which will apply to any of the
+                Thor type components; thoragent, thormanager, thoragent and
+                thorworker, etc.</entry>
+              </row>
+
+              <row>
+                <entry>Target: &lt;name&gt;</entry>
+
+                <entry>The "name" field of each component, typical usage for
+                HPCC Systems components referrs to the cluster name. For
+                example <emphasis>Roxie: -name: roxie</emphasis> which will be
+                the "Roxie" target (cluster). You can also define multiple
+                targets with each having a unique name such as "roxie",
+                "roxie2", "roxie-web" etc.</entry>
+              </row>
+
+              <row>
+                <entry>Pod: &lt;name&gt;</entry>
+
+                <entry>This is the "Deployment" metadata name from the name of
+                the array item of a type. For example, "eclwatch-", "mydali-",
+                "thor-thoragent" This can be a regular expression since
```

Comment:
@xwang2713 - are regular expressions supported or just wildcards?

Reply:
The regular expressions are preferred. But I think  substrings are also OK .  Helm docs (mustRegexMatch):
regexMatch, mustRegexMatch. Returns true if the input string contains any match of the regular expression.